clojure_liberator/src/clojure_liberator/data_access.clj

```diff
@@ -0,0 +1,22 @@
+(ns clojure-liberator.data-access)
+(use 'korma.db)
+(use 'korma.core)
+(require '[clojure.string :as str])
```

Comment:
yeah, I have some cleanup work still to do on this file. I'll add it to this PR tonight. I'm working on making it more idiomatic, based on https://github.com/bbatsov/clojure-style-guide